client/src/store/store.js

```diff
@@ -4,23 +4,29 @@ import userReducer from "./reducers/userReducer";
 import remindsReducer from "./reducers/remindsReducer";
 import { rootSaga } from "./sagas/rootSaga";
 
+import { compose } from "redux";
+
 const testMiddleware = (store) => {
```

Comment:
try to use redux dev tools instead

https://redux.js.org/introduction/ecosystem#dev-tools